redfish-core/lib/log_services.hpp

```diff
@@ -6892,6 +6892,12 @@ inline void fillSystemHardwareIsolationLogEntry(
                     {
                         entryJson["Severity"] = "OK";
                     }
+                    else if (
+                        *severity ==
+                        "xyz.openbmc_project.HardwareIsolation.Entry.Type.Spare")
+                    {
+                        entryJson["Severity"] = "Spare";
```

Comment:
I am almost for sure this will be a Redfish Validator Error. 
Can you run the validator? 
Severity can only be Critical, Warning, or Ok

Reply:
```
                "Severity": {
                    "anyOf": [
                        {
                            "$ref": "#/definitions/EventSeverity"
                        },
                        {
                            "type": "null"
                        }
                    ],
                    "description": "The severity of the log entry.",
                    "longDescription": "This property shall contain the severity of the condition that created the log entry.  If `EntryType` contains `Event`, services can replace the value defined in the message registry with a value more applicable to the implementation.",
                    "readonly": true
                },
 ```
 ```
         "EventSeverity": {
            "enum": [
                "OK",
                "Warning",
                "Critical"
            ],
            "enumDescriptions": {
                "Critical": "A critical condition that requires immediate attention.",
                "OK": "Informational or operating normally.",
                "Warning": "A condition that requires attention."
            },
            
```

Reply:
Yeah. Even we wanted to know how to do this. That is why I just raised a PR to bring up conversation. Thank you